openresty-build-tools/kong-ngx-build

```diff
@@ -18,6 +18,7 @@ OS=
 DIST=
 DIST_VER=
 NGINX_EXTRA_MODULES=()
+PROCESSOR_TYPE=s390x
 
```

Comment:
Added a flag for 2 s390x-specific steps.

Reply:
However, the goal is to be able to remove this flag completely, which we can do if we are able to change the configure script for Linux on Z.

Reply:
removed need for manual s390x setting